### PR TITLE
Scan triggers description activity

### PR DESCRIPTION
**Pull request by Th0h0**
This code implements the whole art description display after a successful user scan.

Right after the user clicks on the scan button (from the ScanFragment), the art's image gets uploaded to Firebase, and its processing (recognition + description fetching) begins. When done, the art's description is directly fed into the art description display activity where the user can actually get a clear insight into the scanned art.

image

### CODE REVIEW

File: app/src/main/java/ch/epfl/culturequest/ui/scan/ScanFragment.java
Comment (TheGuyWithoutH) [inline app/src/main/java/ch/epfl/culturequest/ui/scan/ScanFragment.java:1]: Maybe extract this new code in a method.

**Review by TheGuyWithoutH — COMMENTED** [review]
Seems good, but the code coverage is low, maybe try to augment it.

File: app/src/main/java/ch/epfl/culturequest/ui/scan/ScanFragment.java
Comment (Th0h0) [reply app/src/main/java/ch/epfl/culturequest/ui/scan/ScanFragment.java:1]: You're right. I'll probably rapidly do it during the next sprint.

**Comment by Th0h0** [comment]
> Seems good, but the code coverage is low, maybe try to augment it.

The coverage is better now.

**Comment by Th0h0** [comment]
Waiting for approval.

File: app/src/main/java/ch/epfl/culturequest/ArtDescriptionDisplayActivity.java [hunk 1]
```diff
@@ -0,0 +1,81 @@
...
+        setTextOrFallback(artistNameView, artDescription.getArtist(), "No artist found");
+
+        TextView artYearView = findViewById(R.id.artYear);
+        setTextOrFallback(artYearView, artDescription.getYear(), "No year found");
+
+        TextView artSummaryView = findViewById(R.id.artSummary);
+        setTextOrFallback(artSummaryView, artDescription.getSummary(), "No description found");
+
+        TextView artScoreView = findViewById(R.id.artScore);
+        artScoreView.setText(artDescription.getScore() != null ? artDescription.getScore().toString() : "50");
```
Comment (lucamouchel) [inline app/src/main/java/ch/epfl/culturequest/ArtDescriptionDisplayActivity.java:74]: isnt it risky to put 50 points by default? 50 is a lot no?

**Review by lucamouchel — APPROVED** [review]

**Pull request merged**